featuretools/entityset/serialize.py

```diff
@@ -2,13 +2,16 @@
 import os
 import shutil
 
-from featuretools import variable_types
+from featuretools.utils.gen_utils import find_descendents
+from featuretools.variable_types import Variable
+
+
+def find_variable_types():
```

Comment:
With this new function some variables are becoming redundant.  In this file we have the `VARIABLE_TYPES` variable, which it not referenced in the file, only used in `test_serialization.py`.  In `featuretools/variable_types/variable.py` there's an `ALL_VARIABLE_TYPES` variable which is hard-coded and isn't used at all.

I think we should remove these two variables.  The test cases that depended on `VARIABLE_TYPES` can use this function instead.  I also think it would make more sense for this function to be in `variable.py` instead of `serialize.py`

Reply:
agreed, I'll change all of that